src/main/java/sk/uniba/fmph/dcs/game_board/ToolMakerHutFields.java

```diff
@@ -70,6 +70,17 @@ public boolean canPlaceOnToolMaker(final Player player) {
         return toolMakerFigures.isEmpty();
     }
 
+    public boolean tryToMakeActionToolMaker(final Player player) {
+        return toolMakerFigures.contains(player.playerOrder());
+    }
+
+    public boolean skipActionToolMaker(final Player player) {
+        if (tryToMakeActionToolMaker(player)) {
+            toolMakerFigures.clear();
+        }
+        return true;
```

Comment:
```suggestion
            toolMakerFigures.clear();
            return true;
        }
        return false;
```

Ja to mam ze vrati false ak tam nema figurky, ale asi je to jedno lebo aj tak to ma na starosti GamephaseController. Takto je to mozno aj lepsie aby skip sa vzdy podaril lebo skoro nic nerobi

Reply:
Tak to necham tak ako to mam